src/test/test-seccomp.c

```diff
@@ -216,6 +218,7 @@ static void test_restrict_namespace(void) {
         }
 
         assert_se(wait_for_terminate_and_warn("nsseccomp", pid, true) == EXIT_SUCCESS);
+#endif
```

Comment:
I'm not sure we should skip `test_restrict_namespaces` entirely.
Maybe, we should check that `seccomp_restrict_namespaces` doesn't affect `clone/unshare/setns`. Similar to `test_restrict_address_families`